### PR TITLE
Microsoft Teams actions

### LINKED ISSUE #1422: [ACTION] Microsoft Teams actions
**Is there a specific app this action is for?**
Microsoft Teams

**Please provide a link to the relevant API docs for the specific service / operation.**
https://docs.microsoft.com/en-us/graph/api/overview?view=graph-rest-1.0

**Create Channel** | Create a new channel in Microsoft Teams
**Send Channel Message** | Send a message to a team's channel
**Send Chat Message** | Send a message to a team's chat

### PR DESCRIPTION
**Pull request by michelle0927**
**Create Channel** | Create a new channel in Microsoft Teams
**Send Channel Message** | Send a message to a team's channel
**Send Chat Message** | Send a message to a team's chat
**List Channels** | Lists all channels in a Microsoft Team

Resolves #1422

### CODE REVIEW

File: components/microsoft_teams/actions/list-channels/list-channels.mjs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    microsoftTeams,
+    teamId: {
+      propDefinition: [
+        microsoftTeams,
+        "team",
+      ],
+    },
+  },
+  async run({ $ }) {
+
```
Comment (lcaresia) [inline components/microsoft_teams/actions/list-channels/list-channels.mjs:19]:
```suggestion
```

File: components/microsoft_teams/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "homepage": "https://pipedream.com/apps/microsoft_teams",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@microsoft/microsoft-graph-client": "^3.0.2",
+    "isomorphic-fetch": "^3.0.0"
+  }
+}
```
Comment (lcaresia) [inline components/microsoft_teams/package.json:21]:
```suggestion
}

```

File: components/microsoft_teams/package.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "pipedream/microsoft_teams",
+  "version": "0.0.2",
```
Comment (lcaresia) [inline components/microsoft_teams/package.json:3]:
```suggestion
  "version": "0.0.1",
```

**Review by lcaresia — DISMISSED** [review]
Thanks for your contribution.

Just minor fixes!

**Comment by Hassan-Abbas-SQA** [comment]
@michelle0927  Action :Send Chat Message sends a message to a user who has teams account but not to an external user see example attached screenshot 
image
image

**Comment by dannyroosevelt** [comment]
> @dannyroosevelt @michelle0927 Action :Send Chat Message sends a message to a user who has teams account but not to an external user see example attached screenshot , (its already mentioned in teams that this user is outside of organization some features won’t work) so should i move ticket to ready for release? image image

Yea that makes sense and is probably within the user's expectation, but it might make sense to note that in the description for the prop where the recipient is configured. @Hassan-Abbas-SQA can you make that change and push it to the branch?

**Review by jcortes — APPROVED** [review]
Approved!

**Pull request merged**